website/docs/developer-section/libraries/decentralized-renderer/decentralized-renderer-react-components.md

```diff
@@ -0,0 +1,209 @@
+---
```

Comment:
dont merge btw, need to list down all pages that moved

Reply:
ok, i have created a google sheets and listed what the new routes of the restructured documentation are when compared to the old one. the link is also in the comments at the top of this PR.

google sheets: https://docs.google.com/spreadsheets/d/1lIG30e3151abDlmURe62IvgEHKV7xgcqwppGXbVX3gg/edit?usp=sharing